web/src/app/projects/repository/repository.component.html

```diff
@@ -10,30 +10,144 @@
+            <p class="heading">Latest Release</p>
+            <h3 class="description">v1.0.0</h3>
+          </mat-card-content>
+          <mat-card-actions>
+            <div class="card-footer" fxLayout="row">
+              <img fxFlex="15%" alt ="webkhushboo" src="https://avatars2.githubusercontent.com/u/10898595?v=4"/>
+              <p fxFlex="85%" class="bottom-line" mat-line> Published: 2 days ago</p>
+            </div>
+          </mat-card-actions>
+        </mat-card>
+      </div>
+      <div fxFlex="24%">
+        <mat-card>
+          <mat-card-content class="dashboard-card">
+            <div class="card-icon control-point">
+              <mat-icon>content_copy</mat-icon>
+            </div>
+            <p class="heading">Milestone last updated</p>
+            <h3 class="description">v1.0</h3>
+          </mat-card-content>
+          <mat-card-actions>
+            <div class="card-footer" fxLayout="row">
+              <img fxFlex="15%" alt ="webkhushboo" src="https://avatars2.githubusercontent.com/u/10898595?v=4"/>
+              <p fxFlex="85%" class="bottom-line" mat-line>State: Open | Open: 4 | Close: 10</p>
+            </div>
+          </mat-card-actions>
+        </mat-card>
+      </div>
+      <div fxFlex="24%">
+        <mat-card>
+          <mat-card-content class="dashboard-card">
+            <div class="card-icon verified-user">
+              <mat-icon>verified_user</mat-icon>
+            </div>
+            <p class="heading">Top Contributor</p>
+            <h3 class="description">octocat</h3>
+          </mat-card-content>
+          <mat-card-actions>
+            <div class="card-footer" fxLayout="row">
+              <img fxFlex="15%" alt ="webkhushboo" src="https://avatars2.githubusercontent.com/u/10898595?v=4"/>
+              <p fxFlex="85%" class="bottom-line" mat-line>Total commits : 136</p>
+            </div>
+          </mat-card-actions>
+        </mat-card>
+      </div>
+      <div fxFlex="24%">
+        <mat-card>
+          <mat-card-content class="dashboard-card">
+            <div class="card-icon info-outline">
+              <mat-icon>control_point</mat-icon>
+            </div>
+            <p class="heading">Latest PR</p>
+            <h3 class="description">1347</h3>
+          </mat-card-content>
+          <mat-card-actions>
+            <div class="card-footer" fxLayout="row">
+              <img fxFlex="15%" alt ="webkhushboo" src="https://avatars2.githubusercontent.com/u/10898595?v=4"/>
+              <p fxFlex="85%" class="bottom-line" mat-line>Found a bug | webkhushboo</p>
+            </div>
+          </mat-card-actions>
+        </mat-card>
+      </div>
+    </div>
+    <div fxLayout="row" fxLayoutGap="20px" class="tab-list">
+      <div fxFlex="24%">
+        <mat-card>
+          <mat-card-content class="dashboard-card">
+            <h3 *ngIf="repository.events && repository.events.length > 0; else eventsOther" class="heading card-icon">Events List</h3>
```

Comment:
Other lists do not have the word `Lists` after them, so I think `Events` alone is fine

Reply:
Sure